PYME/Acquire/SpoolController.py

```diff
@@ -665,10 +672,14 @@ def start_spooling(self, filename=None, stack=None, hdf_comp_level=None,
         subdirectory : str, optional
             Directory within current set directory to spool this series. The
             directory will be created if it doesn't already exist.
+        metadata : dict, optional
```

Comment:
I'm not sure we can actually pass a dict in a url - this might effect `pzf_compression_settings` as well.

Reply:
hmm, the test would seem to suggest we can, still feels a little fragile.

Reply:
as would my using it on scope for a week.